index.php

```diff
@@ -0,0 +1,131 @@
+<?php
+require 'apicall.php';
+/**
+ * Class fetch to fetch the data from API.
+ * 
+ * Class for fetching title,service,image,alias and icon 
+ */
+class fetch
+{
+  public $title;
+  public $service;
+  public $image;
+  public $alias;
+  public $iconArr;
+  /**
+   * fetch constructor.
+   * 
+   * @param string $alias,$title,$service,$image,$iconArr
+   * 
+   * Fetched data will be passed as parameter.
+   */
+  function __construct($title, $service, $image, $alias, $iconArr)
```

Comment:
- mention access specifier for each function/method. 
- Also add type for parameters.
```suggestion
  public function __construct(string $title, string $service, string $image, string $alias, string $iconArr)
```